create/application.go

```diff
@@ -29,11 +31,11 @@ type applicationCmd struct {
 	Wait        bool              `default:"true" help:"Wait until application is fully created."`
 	WaitTimeout time.Duration     `default:"15m" help:"Duration to wait for application getting ready. Only relevant if wait is set."`
 	Git         gitConfig         `embed:"" prefix:"git-"`
-	Size        string            `default:"micro" help:"Size of the app."`
-	Port        int32             `default:"8080" help:"Port the app is listening on."`
-	Replicas    int32             `default:"1" help:"Amount of replicas of the running app."`
+	Size        *string           `help:"Size of the app (defaults to \"${app_default_size}\")."`
+	Port        *int32            `help:"Port the app is listening on (defaults to ${app_default_port})."`
+	Replicas    *int32            `help:"Amount of replicas of the running app (defaults to ${app_default_replicas})."`
 	Hosts       []string          `help:"Host names where the application can be accessed. If empty, the application will just be accessible on a generated host name on the deploio.app domain."`
-	BasicAuth   bool              `default:"false" help:"Enable/Disable basic authentication for the application."`
+	BasicAuth   *bool             `help:"Enable/Disable basic authentication for the applicaton (defaults to ${app_default_basic_auth})."`
```

Comment:
Nit: some consistency would be nice, maybe call it `app` everywhere in the help instead of `application`?

Reply:
I don't have a strong preference on either just that `app` is making the help a bit shorter. Other than that consistency is more important.

Reply:
I replaced all occurrences of 'application' with 'app' in the create command.